internal/translation/audit/conversion_test.go

```diff
@@ -0,0 +1,220 @@
+package audit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
+)
+
+func TestConversionContructor(t *testing.T) {
```

Comment:
```suggestion
func TestConversionConstructor(t *testing.T) {
```

Or maybe even better:

```suggestion
func TestNewAuditConfig(t *testing.T) {
```